scalafix-core/src/main/scala/scalafix/syntax/package.scala

```diff
@@ -70,7 +70,14 @@ package object syntax {
   }
 
   implicit class XtensionToken(token: Token) {
-    def posTuple: (Int, Int) = token.start -> token.end
+    // copy pasted from:
+    // github.com/scalameta/scalafmt/blob/master/scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenOps.scala
```

Comment:
If you press "y" while browsing a file on github.com, the url will expand into something like this 

https://github.com/scalameta/scalafmt/blob/ba319e0aaee6d4e1b30098ad57940c67910c33e9/scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenOps.scala

this url will still work even if the file is edited/moved/removed